src/components/User_main.js

```diff
@@ -1,9 +1,105 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import '../styles/components/User_main.css';
+import axios from 'axios';
+import ProgressBar from './bars/ProgressBar';
+import { useState } from 'react';
+import useInterval from '../hooks/useInterval';
 
 const User_main = () => {
+    const [startTime, setStartTime] = useState({
+        sHour: '22',
+        sMinute: '10',
+        status: 'SLOW',
+        change: true
+    });
+    const setSTime = () => {
+        setStartTime({
+            sHour: '22',
+            sMinute: '10',
+            status: 'SLOW',
+            change: !startTime.change
+        })
+        console.log(startTime);
+    }
+    useInterval(setSTime ,60000);
+
+    const [proceedTime, setProceedTime] = useState({
+        pHour: '',
+        pMinute: ''
+    });
+
+    const proceededTime = () => {
+        const today = new Date();
+
+        let currentHour = today.getHours();
+        const currentMinute = today.getMinutes();
+
+        const startHour = parseInt(startTime.sHour);
+        const startMinute = parseInt(startTime.sMinute);
+
+        if (startHour > currentHour) {
+            currentHour = today.getHours() + 24;
+        }
+
+        const sumCurrent = currentHour * 60 + currentMinute;
+        const sumStart = startHour * 60 + startMinute;
+        const proceeded = sumCurrent - sumStart;
+
+        setProceedTime({
+            pHour: parseInt(proceeded / 60),
+            pMinute: proceeded % 60
+        })
+    }
+
+    useEffect(()=>{
+        proceededTime();
+    },[startTime])
+
+    useEffect(() => {
+        const auth = localStorage.getItem('CL_auth') && localStorage.getItem('CL_auth').toLowerCase();
+        axios.get(`${process.env.REACT_APP_PROXY}/member/${auth}`, {
+            headers: {
+                Authorization: localStorage.getItem('CL_accessToken')
+            }
+        })
+            .then((res) => {
+                console.log(res);
+            })
+            .catch((err) => {
+                console.log(err);
+            })
+    }, []);
```

Comment:
This code only serves to output the result to the console. You will need to delete or edit this maybe.

Reply:
I haven't used it yet, but this is the code I need. I'll fix it.